seminaire-saint-sulpice-ecole-theologie.csl

```diff
@@ -510,8 +509,7 @@
     <label plural="never" prefix="n°" suffix=" " variable="number" form="short"/>
     <text variable="locator" form="short"/>
   </macro>
-  <citation et-al-min="4" et-al-use-first="3" et-al-subsequent-min="3" et-al-subsequent-use-first="1"
-      near-note-distance="1">
+  <citation>
```

Comment:
Did you also mean to remove all the et al conditions here? Those would seem to make sense